symplyphysics/laws/optics/lens_makers_equation_for_thin_lenses.py

```diff
@@ -0,0 +1,44 @@
+from sympy import (Eq, solve)

```

Comment:
Sorry but this law was just recently added: https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/optics/optical_power_from_thin_lens_radius.py